packages/web/package.json

```diff
@@ -31,11 +31,11 @@
     "css:prefix": "postcss --config ./config/postcss.config.js --replace \"dist/css/*.css\" \"!dist/css/*.min.css\"",
```

Comment:
`css:fix:prepare` 😄 (just kidding)

Reply:
prepare **post**css :D

Reply:
Happy hour! For a while, I was actually looking for needed change. Nice try, guys!